qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -74,6 +75,9 @@ def __init__(
                  quantum gates on the feature map circuit which may be tuned. If users intend to
                  tune feature map parameters to find optimal values, this field should be set.
             user_parameters: deprecated keyword alias for training_parameters
+
+        Raises:
+            QiskitMachineLearningError: if invoker supplies both user_parameters and training_parameters.
```

Comment:
See above comment - I don't think. You should never see user_parameters in the algo as it would have been renamed by decorator.

Reply:
roger that.